src/components/organisms/site/Header/Header.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { FC } from 'react'
 import classNames from 'classnames/bind'
-import { Link, useStaticQuery, graphql } from 'gatsby'
-import Img from 'gatsby-image'
+import { Link } from 'gatsby'
+
+import { Band } from '../../../molecules/Band/Band'
```

Comment:
I forget the name of it right now, but there's an NPM plugin so we can avoid `../../../../../../` and just import on file path. It's fine now with a site of this size, but just in case you run into a larger React/Gatsby site.